tests/UtilsTest.php

```diff
@@ -289,6 +289,27 @@ public function provideCalculateBitmaskCases()
         ];
     }
 
+    /**
+     * @group legacy
+     * @expectedDeprecation The message
```

Comment:
can we use `expectDeprecationMessage` method?

Reply:
Not really, this method is actually `expectExceptionMessage` call: https://github.com/sebastianbergmann/phpunit/blob/9.5.4/src/Framework/TestCase.php#L646

Reply:
isn't deprecation an exception under the hood?
also, the annotation is simply calling the method, isn't it ?

Reply:
> isn't deprecation an exception under the hood?

yes, in the other test, but here - without `PHP_CS_FIXER_FUTURE_MODE` on - it's only user deprecation.

Reply:
can you try `expectDeprecation` method of Sf phpunit bridge? https://symfony.com/doc/current/components/phpunit_bridge.html#id1

Reply:
Seems I can.